### ISSUE TITLE
Restore `interpolation_none_vs_nearest` example somewhere else in the docs

### ISSUE THREAD

**Issue by anntzer** [matplotlib/matplotlib#7977]
The `interpolation_none_vs_nearest` example was removed in #7952 as @NelleV pointed out that this really doesn't belong in a gallery (no one will really find that info there).  I generally agree with that sentiment, but still believe this information should be kept somewhere; @jenshnielsen seemed to agree with this point as well.

Having skimmed (very quickly) through the docs, I believe one place this could go is at the end of the image tutorial (http://matplotlib.org/devdocs/users/image_tutorial.html), which already contains a discussion about interpolation methods.

**Comment by tacaswell** [comment]
That example `definitely` should not have been deleted.

**Comment by story645** [comment]
While I think that tutorial is a good place for it, does it make sense to put it as an example in the api docs (or is that overkill?) Or, how is the information not already captured here: http://matplotlib.org/devdocs/examples/images_contours_and_fields/interpolation_methods.html

**Comment by NelleV** [comment]
eh… I so strongly disagree with you…

**Comment by NelleV** [comment]
ping @efiring

**Comment by NelleV** [comment]
Just to be clear, too much documentation is bad documentation. We shouldn't have something in our documentation that explains a tiny tiny detail for 0.01% of our users when we can have an example that covers much broadly the exact same topic. Sure, this example needs some love, but so does the 500 other examples in our gallery. 

Again, I am happy to have a communication expert look this over with us.. We can also ask some people that are directly affected by interpolation methods (ie, image processing people) what they think about that example (I actually already had a discussion about this with some…).

**Comment by story645** [comment]
While I think you're correct in big picture terms @NelleV, I think @anntzer is proposing a valid compromise. Don't make it standalone to chip away at doc bloat, but put it in a place where hopefully the people looking for it will find it.

**Comment by NelleV** [comment]
Adding more explanation to the image tutorial seems reasonable (and to the interpolation methods example docstring) , but I am concerned that @tacaswell is suggesting re-adding the example to the gallery.

**Comment by efiring** [comment]
@NelleV, that example is critical in trying to explain something that is rather hard to understand and likely to puzzle anyone curious about using anything other than the default interpolation method.  I think Tom's point is not that it necessarily has to stay in the gallery, but that it should not have been yanked out of there, potentially getting lost entirely, without being moved to some other location in the docs.

**Comment by NelleV** [comment]
I don't see why it needs a full example to itself when it can as easily be explain in the interpolation_methods.py example. The current example brings absolutely no extra information compared to the other one. (And that's without considering that it is very likely that no one ever looks at that example).

**Comment by story645** [comment]
@efringg it wasn't an autonomous decision though. It was yanked via #7952 with review from 2 developers. Maybe the PR/review policy should me modified for documentation PRs?

**Comment by efiring** [comment]
@story645, It's not a matter of procedure; the procedure is fine.  No reasonable procedure can prevent occasional cases where there are disagreements or second thoughts, in documentation and in code.

**Comment by NelleV** [comment]
Considering I am the one who started this whole debate, I can do another pass on the interpolation methods example to (1) add more text and explanation about the differences; (2) add a pdf version of this plot to show the difference in the case of a svg/pdf backend. 
We can then go from there to see if adding another specific example is necessary.

**Comment by story645** [comment]
@NelleV is there a way to use different test data or something that'll show the difference more? Nobody really reads, so I think it has to be visible in the image.

**Comment by NelleV** [comment]
The bottom image of http://matplotlib.org/_images/interpolation_none_vs_nearest_01.png is the one already in the interpolation method example which I am suggesting adding as a PDF as well (which is going to be a pain, as it cannot be generated easily from the documentation).

Also, in this particular case, a single line would be more effective at describing what's going on with 'none' in the case of PDFs: it is just a question of finding out what interpolation is performed when displaying that image when scaled up. Unfortunately, that information is missing from our documentation, so I'll need to dig it out (i think it is a quadratic interpolation but it's hard to say). The example I removed + the text I copied pasted from it makes the interpolation strategy sound super complicated when it really isn't.

**Comment by anntzer** [comment]
@story645 I took the responsibility of merging the PR despite disagreement (shared by @jenshnielsen) about that specific example because I considered that the rest of the improvements were good, and knew that we could open a separate issue regarding that example (which, I agreed, was not ideally located either, so we may as well take advantage of the opportunity to improve it).

**Comment by story645** [comment]
@anntzer that's sort of my point. Ultimately it was a communal decision. *shrugs* It seems to be sorting itself out well enough. 

@NelleV I have general qualms with that example 'cause  so many of the interpolation methods look like they do the same thing there. I feel like visualizations always trump descriptions (even one liners) in explaining the difference between visualization techniques. But I'm not gonna fight with a one liner being the immediate solution and opening a ticket to improve that doc.

**Comment by dopplershift** [comment]
I agree it doesn't belong as an example--but nobody was dying, no scientific results incorrect with this extra example. Taking the time to move this, rather than delete and potentially lose the information, is the better way.

I think for future PRs like that, nothing should be *removed* if there's not consensus on removing it. I agree it was good to get the PR merge to improve the docs, but nixing the deletion would have been the better course.

**Comment by efiring** [comment]
This issue needs to be addressed, but it shouldn't block a 2.0.1 release, so I am moving the target.
